infra/build/functions/project_sync.py

```diff
@@ -138,7 +138,8 @@ def sync_projects(cloud_scheduler_client, projects):
       try:
         logging.info('Schedule changed.')
         update_scheduler(cloud_scheduler_client, project,
-                         projects[project.name].schedule)
+                         projects[project.name].schedule,
+                         build_project.FUZZING_BUILD_TAG)
```

Comment:
Is there no update needed for the coverage tag?

Reply:
No, since coverage scheduler is fixed and will only build once per day, builds_per_day only refers to fuzzing build so only fuzzing build requires update when schedule is changes.